.github/workflows/coordinator-build-and-publish.yml

```diff
@@ -31,6 +31,7 @@ jobs:
   build-and-publish:
     runs-on: ubuntu-22.04
     name: Coordinator build
+    environment: test-and-builds
```

Comment:
suggestion to the name: `docker-build-and-e2e`

Reply:
Done